components/admin-panel/hooks/useReceipt.ts

```diff
@@ -0,0 +1,94 @@
+import { useState } from 'react';
```

Comment:
One small suggestion is that the organization of the files here. Since `useReceipt` hook is only used in the `InvoicesReceipts.js` and `ReceiptTemplateForm.tsx` maybe to better organization we could have a folder `components/admin-panel/sections/invoice-receipts` within which we have `InvoicesReceipts.js`, `ReceiptTemplateForm.tsx` and `hooks/useReceipt.ts`.

Other than this minor comment everything looks good and I've tested it. 💯

Reply:
@SudharakaP Done